icons/aliases.json

```diff
@@ -473,5 +473,10 @@
     "name": "yveCRV-ETH Pickle Jar",
     "symbol": "yveCRV - ETH",
     "address": "0x5Eff6d166D66BacBC1BF52E2C54dD391AE6b1f48"
+  },
+  {
+    "name": "Curve triCrypto Pool yVault",
+    "symbol": "crvTricrypto",
+    "address": "0xcA3d75aC011BF5aD07a98d02f18225F9bD9A6BDF"
   }
```

Comment:
Similar to 3pool: 
```
  {
    "name": "Curve 3pool yVault",
    "symbol": "yvCurve-3pool",
    "address": "0x9cA85572E6A3EbF24dEDd195623F188735A5179f"
  },
  {
    "name": "Curve 3pool",
    "symbol": "3Crv",
    "address": "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"
  },
  ```
  
=> 
Curve triCryptopool yVault
yvCurve-tricryptopool

=> 
Curve triCryptopool
tricryptopool